### ISSUE TITLE
explore query options for expression databases

### ISSUE THREAD

**Issue by mellybelly** [NCATS-Tangerine/ncats-ingest#22]
We need to pull a series of data to inform which genes to look for variants in.

There is a google doc here for reference,
https://docs.google.com/spreadsheets/d/1yX-5sfrC3vrahf4_k7-5rl4Oqzm853ollIMmUo1PTc0/edit#gid=1185309083

This relates to Set-7.

but essentially we need a gene set based upon significant co-expression of our primary genes (some may have alternate primary symbols):
FANCA, FANCB, FANCC, FANCE, FANCF, FANCG, FANCL, FANCM, FANCD2, FANCI, UBE2T
FANCD1 (BRCA2), FANCJ, FANCN, FANCO, FANCP, FANCQ, FANCR, FANCS, FANCV, FANCU
FAAP100, FAAP24, FAAP20, FAAP16 (MHF1), FAAP10 (MHF2)

@tknijnen has already done some excellent work on this with TCGA and GTEx to do the retrieval
@cmungall related to architecture - this may be a good example of a late binding, but needs documentation and planning

@dnahotline will advise as needed

**Comment by tknijnen** [comment]
Yep, I think TCGA and GDSC are great gene expression resources. Perhaps also, the CCLE and GDSC cancer cell line panels. An important challenge is how to combine the various data sources to nominate FA associated genes. Both for correlation analysis and ontology inference, it would interesting to investigate integration of different datasets; which are more important, how to set cutoffs.

**Comment by tknijnen** [comment]
Check. Just wanted to let you know that I am aiming to build an API that would provide access to the correlation statistics of all genes across all tissues/cancer typres in GTEx and TCGA for the hackaton. I have very little experience with APIs (some experience with using them, no experience building them), but I hope that I can leverage the knowledge in Team Blue (and perhaps other colors such as green and orange) to get something going for the hackaton. I think that this fits with your queries and should also be interested to other teams. Let me know you thoughts and suggestions if any. cheers

**Comment by mellybelly** [comment]
Wonderful! I'm so glad you are doing this.
@cmungall can advise RE APIs as he's been doing a lot to create the unified biolink API https://api.monarchinitiative.org/api/
See doc at https://github.com/monarch-initiative/biolink-api/

**Comment by stevencox** [comment]
@tknijnen, @mjstealey of green put together our exposures api. What exactly you'll need to do is highly dependent on your data but the exposures repo with docs is [here](https://github.com/mjstealey/datatranslator). It may be useful if you're working in a Python environment.
